zp-relayer/validateTx.ts

```diff
@@ -233,7 +231,13 @@ export async function validateTx({ txType, rawMemo, txProof, depositSignature }:
     userAddress = await getRecoveredAddress(txType, nullifier, txData, requiredTokenAmount, depositSignature)
     await checkAssertion(() => checkDepositEnoughBalance(userAddress, requiredTokenAmount))
   }
+  if (txType === TxType.PERMITTABLE_DEPOSIT) {
+    const deadline = (txData as PermittableDepositTxData).deadline
+    await checkAssertion(() => checkDeadline(deadline))
```

Comment:
I think we should also configure some buffer window for checking the deadline. e.g. we should reject transactions, whose deadline will expire in 5 seconds, since it won't be mined in such short time in any case.

I would suggest the following:
* During the initial transaction sending, we should reject transaction with deadline in less than 10 minutes
* During transaction resend, we should reject transaction with deadline in less than 10 seconds

Reply:
Agree. Maybe it is better to create a special env (e.g. `PERMIT_DEADLINE_RESEND_WINDOW`) for this deadline window for resend case? And calculate initial sending window as `PERMIT_DEADLINE_RESEND_WINDOW + 10 min`.

Reply:
Current threshold in library is set to 5 minutes, so let's stick to that value for now, for the initial tx sending check - https://github.com/zkBob/zkbob-client-js/pull/34